woodwork/tests/testing_utils/column_utils.py

```diff
@@ -0,0 +1,12 @@
+from woodwork.utils import import_or_none
+
+ks = import_or_none('databricks.koalas')
+
+
+def convert_series(series, logical_type):
```

Comment:
Added this function because for our column accessor tests we need to convert sample series to type `category` for pandas/dask or `object` for Koalas. Once we have completely eliminated DataTable and DataColumn, we might just be able to update our test fixtures to do this instead and remove this function and all the conversion calls in the test.

Reply:
yeah, I think updating the fixtures to do this later makes sense!